mobile-app/app/screens/AppNavigator/screens/Auctions/screens/PlaceBidScreen.tsx

```diff
@@ -135,128 +134,225 @@ export function PlaceBidScreen(props: Props): JSX.Element {
+                {
+                  amount: minNextBidInToken,
+                  symbol: batch.loan.symbol,
+                }
               )}
-              style={tailwind("mt-5")}
-            />
+            </ThemedTextV2>
+          )}
+          {formState.errors.bidAmount?.type === "hasSufficientFunds" && (
+            <ThemedTextV2
+              light={tailwind("text-red-v2")}
+              dark={tailwind("text-red-v2")}
+              style={tailwind("text-xs pt-2 mx-6 font-normal-v2")}
+              testID="insufficient_balance_text"
+            >
+              {translate("screens/PlaceBidScreen", "Insufficient balance")}
+            </ThemedTextV2>
+          )}
+          {displayHigherBidWarning && formState.isValid && (
+            <ThemedTextV2
+              light={tailwind("text-orange-v2")}
+              dark={tailwind("text-orange-v2")}
+              style={tailwind("text-xs pt-2 mx-6 font-normal-v2")}
+              testID="high_bid_text"
+            >
+              {translate(
+                "screens/PlaceBidScreen",
+                "Your bid is higher than the auction's collateral value of {{prefix}}{{amount}}",
+                { prefix: "$", amount: totalCollateralsValueInUSD }
+              )}
+            </ThemedTextV2>
           )}
 
-          <View style={tailwind(["-mx-4", { "mt-4": Platform.OS !== "web" }])}>
-            <ThemedSectionTitle
-              testID="title_tx_detail"
-              text={translate("screens/PlaceBidScreen", "TRANSACTION DETAILS")}
-            />
-            <InfoRow
-              type={InfoType.EstimatedFee}
-              value={fee.toFixed(8)}
-              testID="text_fee"
-              suffix="DFI"
-            />
-          </View>
+          {new BigNumber(bidAmount || 0).gt(0) && (
+            <ThemedViewV2
+              dark={tailwind("border-gray-700")}
+              light={tailwind("border-gray-300")}
+              style={tailwind("p-5 border-0.5 rounded-lg-v2 mx-1 my-6")}
+            >
+              <NumberRowV2
+                lhs={{
+                  value: translate("screens/PlaceBidScreen", "Transaction fee"),
+                  testID: "bid_txn_fee",
+                  themedProps: {
+                    light: tailwind("text-mono-light-v2-500"),
+                    dark: tailwind("text-mono-dark-v2-500"),
+                  },
+                }}
+                rhs={{
+                  value: fee.toFixed(8),
+                  testID: "bid_txn_text_fee",
+                  suffix: " DFI",
+                }}
+              />
+            </ThemedViewV2>
+          )}
         </View>
 
         <View>
-          <Button
-            label={translate("screens/PlaceBidScreen", "CONTINUE")}
+          {blocksRemaining === 0 && (
+            <ThemedTextV2
+              light={tailwind("text-red-v2")}
+              dark={tailwind("text-red-v2")}
+              style={tailwind("text-red-v2 text-center text-xs mb-4")}
```

Comment:
add font-normal-v2, we'll use font-xxx-v2 for every text in v2

Reply:
Added. Should I add for error messages as well?

Reply:
yes, all v2 UI should use v2 font

Reply:
Got it. Added v2 font to all texts